web/modules/custom/feedback/feedback.routing.yml

```diff
@@ -0,0 +1,33 @@
+feedback.admin_settings:
+  path: '/admin/config/content/feedback'
+  defaults:
+    _form: '\Drupal\feedback\Form\FeedbackSettingsForm'
+    _title: 'Feedback Settings'
+  requirements:
+    _permission: 'administer feedback'
+
+feedback.report:
+  path: '/admin/reports/feedback'
+  defaults:
+    _controller: '\Drupal\feedback\Controller\ReportController::createReportTable'
+    _title: 'List of Feedback'
+  requirements:
+    _permission: 'administer feedback'
```

Comment:
I'm not sure but if you specify permission "access feedback overview" ("access content overview" in this PR) you have to use it here

Reply:
I used this permission 'access feedback overview'